scheduler/src/test/java/org/cloudfoundry/autoscaler/scheduler/rest/ScheduleRestControllerTest.java

```diff
@@ -81,6 +81,8 @@
 	private MockMvc mockMvc;
 
 	private String appId = TestDataSetupHelper.generateAppIds(1)[0];
+	private String guid = TestDataSetupHelper.generateGuid();
+	private String guid1, guid2, guid3;
```

Comment:
The guid 's can be defined in method's scope.I guess the existing usage of appId here can also be refactored. Also can live with single guid variable.